dataOrganization.md

```diff
@@ -0,0 +1,62 @@
+# Firestore Data Organization
+## Users
+### Collections
+* [Feeders](#feeders)
+* [Notifications](#notifications)
+### Document Fields
+```ts
+{
+  active: boolean,
+  appIdentifier: string,
+  email: string,
+  firstName: string,
+  id: string,
+  lastName: string,
+  lastOnlineTimestamp: timestamp,
+  phoneNumber: string,
+  profilePictureURL: string
+} 
+```
+## Feeders
+### Collections
+* [Pets](#pets)
+### Document Fields
+```ts
+{
+  id: string,
+  modelNumber: string,
```

Comment:
Is this the id of the feeder that the pet will be fed from? Should we change this to feederNumber.  Something like have the first feeder a user sets up be 0, the second 1 and so on?

Reply:
I intended for the name to be the human readable identifier. Here the model number would be the manufacturer model number. While there will only be one prototype model for our project, I included this to demonstrate that we could allow for multiple different models of our feeders to exist.